src/server.cpp

```diff
@@ -1489,11 +1489,8 @@ void Server::SendMovement(u16 peer_id)
 	Send(&pkt);
 }
 
-void Server::SendPlayerHPOrDie(PlayerSAO *playersao)
+void Server::ForceSendPlayerHPOrDie(PlayerSAO *playersao)
 {
-	if (!g_settings->getBool("enable_damage"))
-		return;
```

Comment:
I think that's the point of this PR. There is currently no way to kill the player if damage is disabled.